src/components/Tabs/Tabs.driver.ts

```diff
@@ -1,22 +1,22 @@
-import { StylableDOMUtil } from '@stylable/dom-test-kit';
-import style from './Tabs.st.css';
 import { Simulate } from 'react-dom/test-utils';
 
 export const tabsDriverFactory = ({ element }) => {
-  const stylableDOMUtil = new StylableDOMUtil(style);
-
   const getItems: any = () =>
     Array.from(element.querySelector('nav').childNodes);
 
+  const getTabs: any = () =>
+    element.querySelector('[data-hook="scrollable-tabs"]');
```

Comment:
Use constant for dataHooks/data-X names (to be reused in production and testing code)

Reply:
I added a `DATA_HOOKS` constant, but i'm not sure adding the `data-X` names will result in readable code. Can you show me an example maybe?